### PR TITLE
Add cfp type in admin/cfp#show

### LINKED ISSUE #1679: admin/cfps#show does not show the type
When in the show page of a cfp, you are not able to tell which cfp type this is. Type should also be visible.

### PR DESCRIPTION
**Pull request by rahul2240**
@cfp.cfp_type is added to all the files to show the cfp type.
Fixes #1679

### CODE REVIEW

**Comment by rahul2240** [comment]
cfp_booth
cfp_event

@Ana06 Please let me know whether this is right or not.

**Comment by rahul2240** [comment]
cfp_events
screenshot from 2017-09-10 13-40-57
booth

File: app/views/admin/cfps/_booths_cfp.html.haml [hunk 1]
```diff
@@ -1,12 +1,16 @@
 %dt
-  Start Date
+  Type:
+%dd
+  = @cfp.cfp_type.capitalize
+%dt
+  Start Date:
```
Comment (Ana06) [inline app/views/admin/cfps/_booths_cfp.html.haml]: Adding the `:` where it was missed is unrelated to the commit _Add cfp type in admin/cfp#show_ Can you please add it in a different commit? So 2 commits :wink:

Comment (rahul2240) [reply app/views/admin/cfps/_booths_cfp.html.haml]: Sure :smile:

**Comment by rahul2240** [comment]
@Ana06 now ok?

**Comment by rahul2240** [comment]
sorry i didn't knew. Now ok?

**Comment by rahul2240** [comment]
Sorry again @Ana06 , I did rebasing and travis is working now :smile:

**Comment by differentreality** [comment]
Nice work @rahul2240 :tada: 
Thank you :+1:

**Comment by rahul2240** [comment]
@Ana06 can you please close this PR if it is fixed?

**Comment by differentreality** [comment]
Nicely done @rahul2240 :+1:  Thank you! :)

**Pull request merged**